airflow-core/src/airflow/jobs/triggerer_job_runner.py

```diff
@@ -589,7 +589,7 @@ def update_triggers(self, requested_trigger_ids: set[int]):
         cancel_trigger_ids = self.running_triggers - requested_trigger_ids
         # Bulk-fetch new trigger records
         new_triggers = Trigger.bulk_fetch(new_trigger_ids)
-        triggers_with_assets = Trigger.fetch_trigger_ids_with_asset()
+        trigger_ids_with_non_task_associations = Trigger.fetch_trigger_ids_with_non_task_associations()
```

Comment:
How about?
```suggestion
        non_task_triggers = Trigger.fetch_non_task_trigger_ids()
```

(etc)

Reply:
I thought about that earlier as well but I changed it because it may imply that `non_task_triggers` includes all triggers where `task_instance` is `None` (regardless of whether they have an asset or a deadline). Another one I considered was `non_task_active_triggers` but I wasn't sure if calling them "active" is a good idea either because this is before a `workload` is created for them

Reply:
I'm interested in getting this merged asap so that the next PRs for deadlines can be created. If you still strongly believe that we should call it `non_task_triggers`, I'm happy to make that change in a later PR but I'll resolve for now